qiskit/transpiler/passmanager.py

```diff
@@ -417,6 +419,13 @@ def _validate_stages(self, stages: Iterable[str]) -> None:
                 for invalid_stage in invalid_stages[1:]:
                     msg.write(f", {invalid_stage}")
                 raise ValueError(msg.getvalue())
+        if len(tuple(stages)) != len(set(stages)):
+            repeated_stages = {s for s in stages if stages.count(s) > 1}
+            warn(
+                f"Stage names {repeated_stages} provided several times, only first occurances preserved.",
```

Comment:
pylint doesn't like how long this is you can split the string into multiple lines and pass lint.